src/client/client.ts

```diff
@@ -221,24 +224,32 @@ export class HTTPClient {
 
   /**
    * Send a GET request.
-   * @param relativePath - The path of the request.
```

Comment:
isn't it changing v2 client as well? Or /v2/client calls these methods internally? Not sue how it is structured.

Reply:
Every HTTP request goes through these functions. To be clear, we are still supporting the v2 algod and indexer APIs -- JS v3 is upgrading the library version, not the REST API versions

Reply:
This is a purely a js client upgrade, APIs should be stable.